### DISCUSSION TITLE
In the newer version I had this introspection problem

### DISCUSSION THREAD

**Discussion by lucasantonioapto** [rebing/graphql-laravel#1093] [General]
error: `Introspection must provide input type for arguments, but received: [FlexibleFilter].`

From the tests we did on the package in version: 8.6 this type works and in version: 9.1 it is not working and is returning this error.

I validated all my types and there is no error in defining types, everything is ok.

**Comment by mfn** [comment]
The upgrades of graphql-laravel often also include updates to webonyx-graphql, especially when we bumped the major to 9 we brought in a big update of webonyx and with it many improvement and strictness; so "tl;dr" I wouldn't be surprised that you now find issues not seen before.

That said, of course could be an issue in this library too 😅

Since you didn't provide much in terms of code to check, I may suggest a quick hack to use the webonyx built-in schema validator and see what it tells you.

In `\Rebing\GraphQL\GraphQL::schema` after this line:
```php
$schema = $this->buildSchemaFromConfig($schemaConfig);
```
add
```php
dd($schema->validate());
```
If it reports something, I suggest to look into that first.

If not … I would need more details to help.

ps: I was meant to add a console command to easily check this but never got around

**Reply by lucasantonioapto** [reply]
Where would this line be: "\Rebing\GraphQL\GraphQL::schema"

I didn't find it in my project, and I did ctrl + shift + F in PHPStorm and didn't find anything

**Reply by mfn** [reply]
https://github.com/rebing/graphql-laravel/blob/05ca63474b7c5dff0ceb52f4aafdceff41e6b2b5/src/GraphQL.php#L66

**Reply by lucasantonioapto** [reply]
I made this repository with the same real code I'm using, and it causes the same problem.

It already includes docker using mysql so you can run it without problems.

Repository: https://github.com/lucasantonioapto/examples-bugs-graphql

**Reply by mfn** [reply]
If you want to use types for "input", they need to extend `Rebing\GraphQL\Support\InputType`, see https://github.com/rebing/graphql-laravel#input-object .

In your project, `FlexibleFilterType` is a regular type.
